player/video.c

```diff
@@ -59,6 +59,7 @@ static const char av_desync_help_text[] =
 "Audio/Video desynchronisation detected! Possible reasons include too slow\n"
 "hardware, temporary CPU spikes, broken drivers, and broken files. Audio\n"
 "position will not match to the video (see A-V status field).\n"
+"Consider trying `--profile=fast` and/or `--hwdec=auto-safe`, as they may help.\n"
```

Comment:
Totally useless grammar nit but I think (?) you don't need the comma.

Reply:
I have no idea. Seems redundant indeed, removed.